package.json

```diff
@@ -103,5 +103,15 @@
     ".js": {
       "style": "sova"
     }
+  },
+  "husky": {
```

Comment:
But our project already has configs for husky and lint-staged:
- https://github.com/woly-ui/woly/blob/master/.lintstagedrc
- https://github.com/woly-ui/woly/blob/master/.huskyrc

Reply:
Sorry, I did not notice this, but already fixed it